packages/skott/src/modules/resolvers/ecmascript/resolver.ts

```diff
@@ -112,6 +115,16 @@ export function isJavaScriptModule(module: string): boolean {
   );
 }
 
+export function isTypeScriptProject(tsConfigPath: string) {
+  return pipe(
+      Effect.service(FileReaderTag),
+      Effect.flatMap(
+          (fileReader) => Effect.tryPromise(() => fileReader.stats(tsConfigPath))
```

Comment:
You need to grab the current working directory that can drastically change the location from which the tsconfig should be looked up (you can see `readTSConfig` from `path-alias.ts` file.

```suggestion
          (fileReader) => Effect.tryPromise(() => fileReader.stats(path.join(fileReader.getCurrentWorkingDir(), tsConfigPath)))
```

Normally resolving these two points should make the whole test suite pass, I'll follow up :)

Reply:
Oh yes you're right, I assumed the fileReader was doing it, I read it to fast

Reply:
Actually this does not entirely solve the problem test-wise, because `stats` behavior is commonly customized in the tests leveraging dependency injection. I'm not sure that we want to tweak `fileReader.stats` behavior to make it work for that use case. 

Consequently I just added now an `exists` function to the FileReader API (#94), at the beginning I wanted to suggest you that but then I figured out that there was some duplication and imports were not good so I fixed it on my own so that you can focus on this PR and the other one.

If I'm not mistaken if you now rebase and use 

```ts
fileReader.exists(path.join(fileReader.getCurrentWorkingDir(), tsConfigPath))
```

So that this Effect directly returns a boolean and there is no further work to do :)